torch/_dynamo/optimizations/distributed.py

```diff
@@ -41,13 +42,20 @@ def pretty_print_buckets(buckets: List[Bucket]):
         print("Please `pip install tabulate` in order to pretty-print ddp bucket sizes")
 
 
+def fx_name(fqn):
+    parts = fqn.split(".")
+    module = "self_" + "_".join(parts[:-1])
+    attr = parts[-1]
+    return module, attr
+
+
 class DDPOptimizer:
     def __init__(
         self,
         bucket_bytes_cap: int,
         parameters_to_ignore: List[str],
         backend_compile_fn,
-        debug=False,
+        debug=True,
```

Comment:
Is this change intended?

Reply:
oops, i thought i updated this back. thanks!